maquetado/scss/styles.scss

```diff
@@ -30,3 +30,25 @@ body {
     grid-row: 1/3;
   }
 }
+
+@media only screen and (max-width: 1024px){
+  .nav-title{
+    display: none;
+  }
+
+  .tech-area-title{
+    margin: 0;
+    padding: 0 40px;
+  }
+
+  .tech-list{
+    display: block;
```

Comment:
You can just simply change the display of the grid to just 1 column and n rows. That way we keep the display grid and use it everywhere 😄

Reply:
You're right! I didn't think of that at that moment. Already on it